mantidproject/mantid
# Update reflectometry release notes

General improvements to the reflectometry release notes:

- Separate the algorithm notes into their own section and add a reference to which
algorithms have changed.
- Change the screenshot to highlight the most significant user-facing changes.
- Reword several of the items to be more concise and to read better as a whole
document.

**To test:**

- Proof read.
- Build docs-html and check that the release notes render ok.

Refs #27100 and refs #27307

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
